docs/_source/practical_guides/create_dataset.md

```diff
@@ -542,20 +576,30 @@ record = rg.FeedbackRecord(
 
 ::::
 
-###### Add `metadata`
+#### Add records to a dataset
 
-Metadata properties allow you to configure the use of metadata information for the filtering and sorting features available in the UI and Python SDK. You can add metadata to your records as you create them. If you want the metadata to correspond with the metadata properties configured for your dataset, make sure that the key of the dictionary corresponds with the metadata property `name`. When the key doesn't correspond, this will be considered extra metadata that will get stored with the record, but will not be usable for filtering and sorting.
+Once you have a list of configured `FeedbackRecord`s, you can add those to a local or remote `FeedbackDataset` using the `add_records` method.
 
 ```python
-record = rg.FeedbackRecord(
-    fields=...,
-    metadata={"source": "encyclopedia"},
-)
+dataset = rg.FeedbackDataset.from_argilla(name="my_dataset", workspace="my_workspace")
+
+records = [
+    rg.FeedbackRecord(
+        fields={"question": record["instruction"], "answer": record["response"]}
+    )
+    for record in hf_dataset if record["category"]=="open_qa"
+]
+
+dataset.add_records(records)
 ```
 
-##### Update records
+```{note}
+As soon as you add records to a remote dataset, these should be available in the Argila UI. If you cannot see them, try hitting the `Refresh` button on the sidebar.
+```
+
+#### Update records
```

Comment:
@nataliaElv perhaps it is good to reference the formatting headers above or maybe duplicate part of the code/format in the code snippet underneath?